File: examples/js/answer/question/askAQuestion.js
const { AnswerService } = require("m3o/answer");

// Ask a question and receive an instant answer
async function askAquestion() {
  let answerService = new AnswerService(process.env.MICRO_API_TOKEN);
  let rsp = await answerService.question({
    query: "microsoft",
  });
  console.log(rsp);
}

askAquestion();

File: examples/go/answer/question/askAQuestion.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/answer"
)

// Ask a question and receive an instant answer
func AskAquestion() {
	answerService := answer.NewAnswerService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := answerService.Question(&answer.QuestionRequest{
		Query: "microsoft",
	})
	fmt.Println(rsp, err)
}
